Scripts/99 SL-ThemePrefs.lua

```diff
@@ -191,7 +191,7 @@ local SL_CustomPrefs =
 	DefaultGlobalOffsetSeconds = {
 		-- The default value should not make it to gameplay, set to something nice
```

Comment:
Thanks for the fix! Would you mind removing this comment?

Reply:
Oops, yeah that's a good idea isn't it. I've removed it and edited the comment above it to reflect the fact that the value is used in certain circumstances.

Reply:
Awesome stuff, thanks!